capstone/src/main/webapp/autocomplete.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Autocomplete
+ * @param {inp}
+ * @arg {arr}
+ * @return {void}
+ * Attributing this code to: w3 schools
+ * Link: https://www.w3schools.com/howto/howto_js_autocomplete.asp
+*/
+function autocomplete(inp, arr) {
+  let currentFocus;
+  inp.addEventListener('input', function(e) {
+    let a = this.value;
+    let b = this.value;
+    let i = this.value;
+    let val = this.value;
+    closeAllLists();
+    if (!val) { 
+        return false;
+    }
+    currentFocus = -1;
+    a = document.createElement('DIV');
```

Comment:
Instead of assigning `this.value` to `a`, you can make `a` `const` here since you don't assign elsewhere.

Reply:
thanks! will do!